rust/src/transformations/make_stable_expr/namespace_dt/expr_datetime_component/test.rs

```diff
@@ -0,0 +1,33 @@
+use crate::metrics::SymmetricDistance;
+use crate::transformations::make_stable_expr::namespace_dt::test::{
+    example_dt_lf, parse_example_dt_lf,
+};
+use crate::transformations::make_stable_lazyframe;
+
+use super::*;
+
+#[test]
+fn test_make_expr_components() -> Fallible<()> {
+    let (lf_domain, lf) = example_dt_lf()?;
+    let lf_plan = parse_example_dt_lf(lf.clone()).with_columns([
+        col("time").dt().second(),
+        col("datetime").dt().ordinal_day(),
+        col("date").dt().quarter(),
```

Comment:
```suggestion
        col("time").dt().second().alias("second"),
        col("datetime").dt().ordinal_day().alias("day"),
        col("date").dt().quarter().alias("quarter"),
```
so the headers on the table make sense?